R/bs-theme.R

```diff
@@ -377,11 +377,14 @@ bootstrap_bundle <- function(version) {
     ),
     # nav_spacer() CSS (can be removed)
     nav_spacer = sass_layer(
-      rules = sass_file(system_file("nav-spacer/nav-spacer.scss", package = "bslib"))
+      rules = sass_file(path_bslib_component("scss", "nav-spacer.scss"))
     )
   )
 }
 
+path_bslib_component <- function(...) {
+  system.file("components", ..., package = "bslib")
```

Comment:
```suggestion
  system_file("components", ..., package = "bslib")
```

Reply:
I'm also a little tempted to say that we should abstract and use the current `lib_path()` for this (to get file checking) and call it `inst_path()`, then `lib_path()` and `component_path()` could be implemented as:

```r
lib_path <- function(...) {
  inst_path("lib", ...)
}
components_path <- function(...) {
  inst_path("components", ...)
}
```

Reply:
I did this by abstracting the existing `lib_file()` into `inst_file()` and then adding `lib_file()` and `components_file()`.

I like the approach but not the names. I'd prefer `path_inst()`, `path_lib()`, `path_components()`, etc. which would help with internal discoverability.